pkg/controller/dataimportcron-controller.go

```diff
@@ -57,7 +57,7 @@ import (
 	"kubevirt.io/containerized-data-importer/pkg/common"
 	cc "kubevirt.io/containerized-data-importer/pkg/controller/common"
 	cdv "kubevirt.io/containerized-data-importer/pkg/controller/datavolume"
-	"kubevirt.io/containerized-data-importer/pkg/monitoring/metrics/cdi-controller"
+	metrics "kubevirt.io/containerized-data-importer/pkg/monitoring/metrics/cdi-controller"
```

Comment:
isn't the package already named metrics? is this nicer?

Reply:
Right. No clue why my VS go formatter insists to add it only for this specific import. Any guess?

Reply:
Same for me, let's keep it so we're not breaking some consensus

Reply:
I think the problem is that this package is not named `cdi-controller`, it's named `metrics`
"kubevirt.io/containerized-data-importer/pkg/monitoring/metrics/cdi-controller"

Reply:
cc @avlitman not sure if this is the intention